exonum/src/blockchain/schema.rs

```diff
@@ -92,8 +93,8 @@ where
     }
 
     /// Returns table that keeps a list of transactions for the each block.
-    pub fn block_txs(&self, height: u64) -> ProofListIndex<&T, Hash> {
-        ProofListIndex::new(gen_prefix(CONSENSUS, 4, &height), &self.view)
+    pub fn block_txs(&self, height: Height) -> ProofListIndex<&T, Hash> {
+        ProofListIndex::new(gen_prefix(CONSENSUS, 4, &height.0), &self.view)
```

Comment:
Unfortunately, no: `gen_prefix` is a generic function, that works with different "suffixes".